init.lua

```diff
@@ -109,21 +109,11 @@ minetest.register_craft({
 
 -- Chiseled Quartz
 minetest.register_craft({
-	output = 'quartz:chiseled 2',
+	output = 'quartz:chiseled 4',
 	recipe = {
-		{'stairs:slab_quartzblock', '', ''},
-		{'stairs:slab_quartzblock', '', ''},
-		{'', '', ''},
-	}
-})
-
--- Chiseled Quartz (for stairsplus)
-minetest.register_craft({
-	output = 'quartz:chiseled 2',
-	recipe = {
-		{'quartz:slab_block', '', ''},
-		{'quartz:slab_block', '', ''},
-		{'', '', ''},
+		{'quartz:block', 'quartz:block', ''},
+		{'quartz:block', 'quartz:block', ''},
+		{'',             '',             ''},
```

Comment:
Why not
```
{'quartz:block', 'quartz:block'},
{'quartz:block', 'quartz:block'}
```
?

Reply:
Probably because I was following the pattern of the other crafting recipes in this mod. Would you prefer I either
1. change just this recipe to a 2-by-2 table
2. change all the n-by-m recipes to a n-by-m table
3. just leave it as it is?

Reply:
Oh. I see the mod uses the same style for all other recipes. Fine if you keep it as-is :)